### ISSUE TITLE
OSA Meeting Friday Oct 2nd 2020

### ISSUE THREAD

**Issue by mattodd** [opensourceantibiotics/Series-2-Diarylimidazoles#30]
Meeting Oct 2nd 2020 at 2pm UK time at https://ucl.zoom.us/j/92800004715. This page follows on from #29.

Recording is [here](https://youtu.be/a9CruFGuJtY).
On the call:  @mattodd @edwintse @danaklug @bendndi @MFernflower 

1. **Mechanism of action**: @danaklug checked in with Lee Graves, who provided this update (Sept 28th):
"We received the compounds from Carrow (thanks Carrow!). Mat suggested we test the compounds before doing the MIBs proteomics.  So – I contacted a lab here at UNC (Brian Conlon) doing MRSA research.  His graduate student Jenna Beam ran a quick MIC viability assay.  Compound 26 showed efficacy greater that 50 uM, whereas compound 28 showed efficacy at ~10 um.  Jenna also prepared MRSA cell pellets for us to do the proteomics experiments.  So the plan next week is to do the lysis and drug incubation experiments followed by the MIBs proteomics analyses.  Our end of things will take about a week- UNC proteomics is a little back-logged (about 3wk turn around)- so hopefully by the end of the month we will have data to show."
...which sounds awesome.

2. **in vitro PK**: Monash data discussed in #29. 
a) Residual uncertainty about the units between original data and those from Monash. Are they comparable?
b) Implication: the benzothiophene is something that maybe should be avoided, and if we want to carry out ID studies on metabolites maybe we should try to discern an alternative that possesses some activity without the benzothiophene moiety. @drc007 suggested the **benzofuran compound (821)**. _This was agreed to be a good idea, and will influence the chemistry done next._
c) Met ID studies with Monash or Hypha need to factor this in. **Action**: @danaklug and @edwintse we can wait for data on new analogs incoming, but do we have stock of 821 if that compound is chosen for metID?

3. **MRSA and tox screening**.
MRSA: Next screen in Oct, when all compounds have arrived from #27.
Tox: Important to do given comments on selectivity index from @bendndi [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/25#issuecomment-691936520). Mat in discussions. **Action** on Mat to finalise.

4. **Biofunctionalisation**: **Action** still on Mat to report about ongoing discussions.

5. **Series Origin/Similarity Landscape**: Mat is awaiting clearance for the GSK conversation and will report back when that's received.

6. **Community updates**: Need 30-second video update using the Tha/Dana protocol (OpenSourceMalaria/TechOps#3) describing first MRSA results, to highlight new injections of compounds from @bendndi and collaborators, and to highlight new data from Monash. Action on @danaklug @edwintse to do a 30-second thing we can post.

7. Gram+/Gram- results. Mat re-contacted @TwistedSwisster to track down original CO_ADD report and to try direct approach to CO_ADD. Question remains: are these compounds active against other gram +ves? Awaiting response.

8. Compound Donations. Compounds have arrived? @danaklug to update. Just waiting on Imperial compounds. @danaklug to chase since it'd be very nice to have these by the time of the next screen (in one week).

9. Synthetic Chemistry Update by @edwintse and @danaklug 
Dana to blitz troublesome coupling with microwave and deeplitdive on related compounds.
Ed to complete 3- and 4-pyridyl compounds, sulfone and sulfoxide and route to pyrimidine.

AOB:
11. Note: the informal codes we're using - are they aligned with the formal numbering system?
12. Ask Paul Stapleton to compare the MRSA potency methods he used with the COADD results and comment on any significant differences. **Action** on @danaklug Done. Awaiting response.
13. Are we likely to see any metabolic clearance in the MRSA potency assay? This seems unlikely, which would argue against a metabolite being responsible for the in vitro potency. **Action** @danaklug to check with Paul S. Done, awaiting response.

**From meeting:**
14) discussion of the chemistry (below - please post the two slides @edwintse). @bendndi suggested looking into one-pot Negishi couplings.
15) @bendndi also suggested looking into this cycloaddition (whether it's known), if I understood the suggestion correctly? 

<a href="url"><img src="https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Meetings/Oct%202020/Ben%20Perry%20DNDi%20Suggestion.png?raw=true" align="centre" height="190" ></a>

16) @MFernflower suggested, as a variant of the known compounds OSA823 and OSA824 (see [SAR page](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Potency-Evaluation-and-SAR)) that we look into something like p-OCHF2, and he agreed to look into relevant starting materials from e.g. Fluorochem.

<a href="url"><img src="https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/blob/master/Meetings/Oct%202020/Anthony%20Sama%20OCHF2%20Suggestion.png?raw=true" align="centre" height="190" ></a>

Actions other than the chemical targets discussed (can be checked as complete after the meeting).

- [ ]  ACTION ITEM: Mat to report on possibility of SoP-based tox screening.
- [ ]  ACTION ITEM: Mat to report back on collaboration possibilities with Hypha.
- [ ]  ACTION ITEM: Mat to report back from GSK on what can be shared publicly about the series history.
- [x]  ACTION ITEM: Mat to chase Bill Zuercher about original CO-ADD screening report.
- [x]  ACTION ITEM: Mat This meeting video to be posted on YouTube.
- [ ]  ACTION ITEM: Ed and Dana to make 30-second video highlighting the new project inputs.
- [ ]  ACTION ITEM: Dana to chase donated Imperial compounds.

**Comment by drc007** [comment]
Can't make the meeting this week, let me know if you need me to do anything.

**Comment by MFernflower** [comment]
@mattodd @danaklug @edwintse 

http://www.fluorochem.co.uk/Products/Product?code=004836

4-(Difluoromethoxy)bromobenzene is very cheap 

(10 grams for 13 GBP)

**Comment by MFernflower** [comment]
Cheap precursors for my proposed compounds: 

@mattodd @danaklug @edwintse 

http://www.fluorochem.co.uk/Products/Product?code=004836
http://www.fluorochem.co.uk/Products/Product?code=013041
https://www.sigmaaldrich.com/catalog/product/aldrich/111414?lang=en&region=GB

**Comment by edwintse** [comment]
![Untitled Wiley Document-9](https://user-images.githubusercontent.com/18062981/95563971-0424a200-0a16-11eb-986c-bbb04e396279.jpeg)

**Comment by danaklug** [comment]
Link to this Issue has been created in "Project Meetings" wiki.
